network/80-wifi-adhoc.network

```diff
@@ -17,3 +17,4 @@ WLANInterfaceType=ad-hoc
 
 [Network]
 LinkLocalAddressing=yes
+IPv6AcceptRA=no
```

Comment:
hmm, in order to match 80-auto-link-local.network we probably should add MulticastDNS=yes on this one, ad-hoc wlan is kinda the same story.